app/src/main/java/ch/hikemate/app/ui/map/MapScreen.kt

```diff
@@ -339,47 +339,50 @@ fun CollapsibleHikesList(hikingRoutesViewModel: ListOfHikeRoutesViewModel, isSea
   val routes = hikingRoutesViewModel.hikeRoutes.collectAsState()
   val context = LocalContext.current
 
+  // Map of route ID to the elevation data of the route
+  val elevationDatas = remember { mutableStateOf(emptyMap<String, List<Double>>()) }
+
   // BottomSheetScaffold adds a layout at the bottom of the screen that the user can expand to view
   // the list of hikes
   BottomSheetScaffold(
       scaffoldState = scaffoldState,
       sheetContainerColor = MaterialTheme.colorScheme.surface,
       sheetContent = {
         Column(modifier = Modifier.fillMaxSize().testTag(MapScreen.TEST_TAG_HIKES_LIST)) {
-          LazyColumn(modifier = Modifier.fillMaxSize()) {
-            if (isSearching) {
-              item {
-                Column(
-                    horizontalAlignment = Alignment.CenterHorizontally,
-                    modifier = Modifier.fillMaxWidth()) {
-                      Text(
-                          text = "Searching for hikes...",
-                          style = MaterialTheme.typography.bodyLarge,
-                          textAlign = TextAlign.Center,
-                          modifier =
-                              Modifier.padding(bottom = 16.dp)
-                                  .testTag(MapScreen.TEST_TAG_SEARCHING_MESSAGE))
-                      CircularProgressIndicator(
-                          modifier = Modifier.testTag(MapScreen.TEST_TAG_SEARCH_LOADING_ANIMATION))
-                    }
-              }
-            } else if (routes.value.isEmpty()) {
-              item {
-                // Use a box to center the Text composable of the empty list message
-                Box(modifier = Modifier.fillMaxWidth(), contentAlignment = Alignment.Center) {
+          if (isSearching) {
+            Column(
+                horizontalAlignment = Alignment.CenterHorizontally,
+                modifier = Modifier.fillMaxWidth()) {
                   Text(
-                      text = context.getString(R.string.map_screen_empty_hikes_list_message),
+                      text = "Searching for hikes...",
```

Comment:
Nitpick: Consider saving this string as a string resource value for better maintainability and reusability

Reply:
I didn't do that (it was part of the MapScreen PR) but I fixed it anyway

Reply:
thanks